### PR TITLE
KFSPTS-11478: Upgrade to 2018-03-15 version of financials

### PR DESCRIPTION
**Pull request by cah292**
Please do NOT merge this yet; I still need to validate the code locally.

### CODE REVIEW

File: src/main/java/org/kuali/kfs/kns/util/WebUtils.java [hunk 1]
```diff
@@ -761,7 +761,7 @@ public static void reuseErrorMapFromPreviousRequest(KualiDocumentFormBase kualiD
      * @return
      */
     public static String filterHtmlAndReplaceRiceMarkup(String inputString) {
-        String outputString = StringEscapeUtils.escapeHtml(inputString);
+        String outputString = StringEscapeUtils.escapeHtml4(inputString);
```
[the remark this answers is not in the document]
Comment (cah292) [reply src/main/java/org/kuali/kfs/kns/util/WebUtils.java]: This is an overlay of an existing KFS file, and the base KFS code is using the "escapeHtml4" method here, so I believe that is the correct one to use.

**Comment by cah292** [comment]
With regards to the imports cleanup:

Some of the cu-kfs files with lang/collections/etc. changes also had unused imports , so I went ahead and cleaned some of those up since I was changing the needed imports anyway. There were a few cases where the existing lang/collections/etc. references were unused imports to begin with, so I did an import-cleanup operation that removed the other unused imports as well.

I was just trying to clean up those unused imports since I was impacting other imports in those files anyway, though I do agree that a separate user story would make more sense to do this on a larger scale. If you think the changes to imports apart from the lang/collections/etc. ones should be undone, please let me know.

**Comment by jhulslander** [comment]
Thanks for the clarification on both questions.  I think it makes sense to keep the clean up on the classes that have been done.

**Comment by cah292** [comment]
I've finished my local testing successfully. This can be merged once it has been approved.

**Pull request merged**